content/sensu-go/5.11/installation/install-sensu.md

```diff
@@ -5,385 +5,341 @@ description: "Sensu Go is available for Linux, Windows (agent and CLI only), mac
-
-{{< platformBlockClose >}}
+{{< highlight "RHEL/CentOS" >}}
+# Copy the config template from the docs
+sudo curl -L https://docs.sensu.io/sensu-go/latest/files/backend.yml -o /etc/sensu/backend.yml
 
-{{< platformBlock "Windows" >}}
-
-#### Windows
-
-Copy the example agent config file from `%ALLUSERSPROFILE%\sensu\config\agent.yml.example` (default: `C:\ProgramData\sensu\config\agent.yml.example`) to `C:\ProgramData\sensu\config\agent.yml`.
+# Start sensu-backend using a service manager
+sudo service sensu-backend start
 
-{{< highlight text >}}
-cp C:\ProgramData\sensu\config\agent.yml.example C:\ProgramData\sensu\config\agent.yml
+# Verify that the backend is running
+sudo service sensu-backend status
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
+{{< /language-toggle >}}
 
-{{< platformBlock "Ubuntu/Debian RHEL/CentOS Windows" >}}
+For a complete list of config options, see the [backend reference][6].
 
-### 3. Start the service
+### 3. Open the Sensu web UI
 
-{{< platformBlockClose >}}
+The web UI provides a unified view of your monitoring events with user-friendly tools to reduce alert fatigue.
+After starting the Sensu backend, open the web UI by visiting http://localhost:3000.
+You may need to replace `localhost` with the
+hostname or IP address where the Sensu backend is running.
 
-{{< platformBlock "Ubuntu/Debian RHEL/CentOS" >}}
+To log in, enter your Sensu user credentials, or use Sensu's default admin credentials: username: `admin` and password: `P@ssw0rd!`.
+For now, you should see a blank slate: no incidents or entities.
+Select the ☰ icon to explore the web UI.
 
-#### Linux
+### 4. Make a request to the Sensu health API
 
-Start the agent using a service manager.
+To make sure the backend is up and running, we'll create check the health of the backend using the Sensu API.
 
 {{< highlight shell >}}
-sudo service sensu-agent start
-{{< /highlight >}}
-
-Verify that the agent is running.
-
-{{< highlight shell >}}
-service sensu-agent status
-{{< /highlight >}}
-
-{{< platformBlockClose >}}
-
-{{< platformBlock "Windows" >}}
-
-#### Windows
-
-Change to the `sensu\sensu-agent\bin` directory where you've installed Sensu.
-
-{{< highlight text >}}
-cd 'C:\Program Files\sensu\sensu-agent\bin'
-{{< /highlight >}}
-
-Run the `sensu-agent` executable.
-
-{{< highlight text >}}
-./sensu-agent.exe
+curl http://127.0.0.1:8080/health
 {{< /highlight >}}
 
-Run the following command to install and start the agent.
-
-{{< highlight text >}}
-./sensu-agent service install
-{{< /highlight >}}
-
-Verify that the agent is running.
-
-{{< highlight text >}}
-sc.exe query SensuAgent
-{{< /highlight >}}
-
-{{< platformBlockClose >}}
-
-{{< platformBlock "Ubuntu/Debian RHEL/CentOS Windows" >}}
-
-### Next steps
+You should see a 200 OK response from the API and `"Healthy": true` if the backend is up and running.
```

Comment:
These might both be nitpicks, but you don't actually see the return code with the current `curl` command. Do we want to add the flag that shows the header? I suppose there is an assumption there if it does return that it is a 200 OK. 

Also, more than just `"Healthy": true` is outputted. Maybe say the output will include this message if the idea here is to keep the install guide as minimal as possible vs showing the entire output, which will differ anyways (cluster_id, member_id).

Reply:
Fixed in https://github.com/sensu/sensu-docs/pull/1637/commits/06ae7852ea0ff15d352207efccaa9a2e5bd4d38c